### DISCUSSION TITLE
TypeError: (0 , Kz.removeDB) is not a function || HELP !!!

### DISCUSSION THREAD

**Discussion by YoiRida** [w-okada/voice-changer#1214] [Q&A]
Voice Changer Version
MMVCServerSIO_win_onnxgpu-cuda_v.1.5.3.17b

Operational System
Windows 11

GPU
RTX3060

Read carefully and check the options
 I've tried to Clear Settings
 Sample/Default Models are working
 I've tried to change the Chunk Size
 GUI was successfully launched
 I've read the [tutorial](https://github.com/w-okada/voice-changer/blob/master/tutorials/tutorial_rvc_en_latest.md)
 I've tried to extract to another folder (or re-extract) the .zip file
Model Type
RVC

Issue Description
When starting the program, the program tries to load the index file, but cannot find it.

Booting PHASE :main
PYTHON:3.10.11 (tags/v3.10.11:7d4cc5a, Apr 5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)]
Activating the Voice Changer.
[Voice Changer] download sample catalog. samples_0004_t.json
[Voice Changer] download sample catalog. samples_0004_o.json
[Voice Changer] download sample catalog. samples_0004_d.json
[Voice Changer] model_dir is already exists. skip download samples.
Internal_Port:18888
protocol: HTTP
-- ---- --
Please open the following URL in your browser.
http://:/
In many cases, it will launch when you access any of the following URLs.
http://127.0.0.1:18888/

[VCClient] Access http://127.0.0.1:18888/
[VCClient] wait web server...0 http://127.0.0.1:18888/
[Voice Changer] generate new embedder. (no embedder)
[Voice Changer] Loading index...
[Voice Changer] Index file is not found

The program starts without a problem after that, but when I try to enter an input method, this error code pops up.

TypeError: (0 , Kz.removeDB) is not a function
at http://127.0.0.1:18888/index.js:2:3264601
at m (http://127.0.0.1:18888/index.js:2:1507737)
at Generator. (http://127.0.0.1:18888/index.js:2:1509083)
at Generator.next (http://127.0.0.1:18888/index.js:2:1508166)
at e (http://127.0.0.1:18888/index.js:2:1514937)
at c (http://127.0.0.1:18888/index.js:2:1515140)

I tried several ways to solve the problem, but nothing worked.

Application Screenshot
Captura de pantalla 2024-01-28 014306
Captura de pantalla 2024-01-28 014337

Logs on console
Booting PHASE :main
PYTHON:3.10.11 (tags/v3.10.11:7d4cc5a, Apr 5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)]
Activating the Voice Changer.
[Voice Changer] download sample catalog. samples_0004_t.json
[Voice Changer] download sample catalog. samples_0004_o.json
[Voice Changer] download sample catalog. samples_0004_d.json
[Voice Changer] model_dir is already exists. skip download samples.
Internal_Port:18888
protocol: HTTP
-- ---- --
Please open the following URL in your browser.
http://:/
In many cases, it will launch when you access any of the following URLs.
http://127.0.0.1:18888/

[VCClient] Access http://127.0.0.1:18888/
[VCClient] wait web server...0 http://127.0.0.1:18888/
[Voice Changer] generate new embedder. (no embedder)
[Voice Changer] Loading index...
[Voice Changer] Index file is not found

**Comment by AnnoyingFox** [comment]
The voice models included have no index file with them. Some voices includes an index file among the model itself. Or in other words, the index file will only load if you provide one. This file is not actually necessary for the voice models to work, it just adds like "accent" to put it in some way. It helps when you don't only want to sound like the model but also have the same way of speaking.
